manifests/params.pp

```diff
@@ -47,7 +47,8 @@
       $apache_wsgi_socket_prefix = '/var/run/apache2/wsgi'
       $apacheconf_dir            = '/etc/apache2/sites-available'
       $apacheports_file          = 'ports.conf'
-      $service_provider          = undef
+      $apache_logdir_graphite    = '/var/log/apache2/graphite-web'
+      $service_provider          = $::service_provider
```

Comment:
All CI tests for Debian and Ubuntu are failing, because of the unknown variable $::service_provider.